.github/workflows/im-build-test-deploy.yaml

```diff
@@ -96,10 +97,16 @@ jobs:
           echo "CLASSIFICATION=prod" >> $GITHUB_ENV
 
       - name: "Environment: feature"
-        if: startsWith(github.ref, 'refs/heads/feature/')
+        if: github.event_name == 'pull_request'
         run: |
-          echo "ENVIRONMENT=feature" >> $GITHUB_ENV
-          echo "CLASSIFICATION=dev" >> $GITHUB_ENV
+          # convert to lower case
+          HEAD_BRANCH_LOWERCASE=${GITHUB_HEAD_REF,,}
+          # substitute all forward slashes "/" with hyphens "-"
+          HEAD_BRANCH_LOWERCASE_NO_SLASHES=${HEAD_BRANCH_LOWERCASE//\//-}
+
+          echo "ENVIRONMENT=$HEAD_BRANCH_LOWERCASE_NO_SLASHES" >> $GITHUB_ENV
+          echo "CLASSIFICATION=feature" >> $GITHUB_ENV
```

Comment:
I feel the same for classification `feature` as for the label. Would we use this classification for anything other then deployments triggered via github? If not we could name it `github`.

Reply:
The `ENVIRONMENT` and `CLASSIFICATION` variables are used to distinguish between the separate "feature" environments that are deployed within the same `instance-manager-feature` k8s namespace.

Reply:
I get that, its just that `feature` does not seem right as it does not only related to features, hence my question 🤔 

> Would we use this classification for anything other then deployments triggered via github? If not we could name it github.

Reply:
Yeah, we'd still use that classification for our own personal environments `ivo`, `tons` and `radnov`.